examples/webpack.config.local.js

```diff
@@ -8,43 +8,154 @@ const {resolve} = require('path');
 const ALIASES = require('ocular-dev-tools/config/ocular.config')({
   root: resolve(__dirname, '..')
 }).aliases;
+const ROOT_DIR = resolve(__dirname, '..');
+const LIB_DIR = resolve(__dirname, '..');
```

Comment:
If they are same, is it possible to just consolidate into one?

Reply:
This came over from deck.gl. @Pessimistress do you know if keeping two directory variables helps to clarify something, or have some other benefit?